test/api/fetchArtifactTests.js

```diff
@@ -0,0 +1,33 @@
+const proxyquire = require('proxyquire');
+const sinon = require("sinon");
+const chai = require("chai");
+const assert = chai.assert;
+
+const auth = {
+  url: "test",
+  user: "user",
+  password: "password"
+};
+
+const dataMock = {artifacts:[{relativePath:"test"}]};
+const JenkinsWrapper = {
+    build: {
+        get: sinon.stub().yields(null, dataMock)
+    }
+};
+
+const jenkinsStub = () =>
+     JenkinsWrapper;
```

Comment:
Here is a matter of cleanness.
`const jenkinsStub = () => JenkinsWrapper;`
Reads better than
```
const jenkinsStub = function() {
    return JenkinsWrapper;
}
```